src/Api/Controllers/OrganizationExportController.cs

```diff
@@ -41,8 +41,8 @@ public async Task<IActionResult> Export(Guid organizationId)
         IEnumerable<Collection> orgCollections = await _collectionService.GetOrganizationCollections(organizationId);
         (IEnumerable<CipherOrganizationDetails> orgCiphers, Dictionary<Guid, IGrouping<Guid, CollectionCipher>> collectionCiphersGroupDict) = await _cipherService.GetOrganizationCiphers(userId, organizationId);
 
-        // Backward compatibility with versions 2022.9.0 and 2022.10.0 that use ListResponseModel
-        if (new[] { "2022.9.0", "2022.10.0" }.Contains(_currentContext.ClientVersion))
+        // Backward compatibility with versions between 2022.9.0 and 2022.11.0 that use ListResponseModel
+        if (_currentContext.ClientVersion >= new Version("2022.9.0") && _currentContext.ClientVersion < new Version("2022.11.0"))
```

Comment:
I don't think we need to check that version is greater than 9 since previous versions don't use this api.

Reply:
Nicely spotted! I've changed it to only check if the version is inferior to `2022.11.0`.